lib/site_prism/page.rb

```diff
@@ -72,16 +72,18 @@ def template_backed_matches
       matcher_template.mappings(page.current_url)
     end
 
+    # rubocop:disable Naming/AccessorMethodName
```

Comment:
lets just leave this as a regular rubocop override. As we might provide other `setters` later on in the projects life.